packages/remix/src/generators/setup-tailwind/__snapshots__/setup-tailwind.impl.spec.ts.snap

```diff
@@ -31,8 +31,8 @@ exports[`setup-tailwind generator should add a js tailwind config to an applicat
   Scripts,
   ScrollRestoration,
 } from '@remix-run/react';
-import styles from './tailwind.css';
-export const links = () => [{ rel: 'stylesheet', href: styles }];
+import tw_styles from './tailwind.css';
```

Comment:
Can we keep it mixed case since it's JS code?

Reply:
Fixed